python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -98,3 +99,47 @@ private module Re {
     override DataFlow::Node getRegexNode() { result = regexNode }
   }
 }
+
+// ---------------------------------------------------------------------------
+// xmltodict
+// ---------------------------------------------------------------------------
+/** Gets a reference to the `xmltodict` module. */
+API::Node xmltodict() { result = API::moduleImport("xmltodict") }
+
+/**
+ * A call to `xmltodict.parse`
+ * See https://github.com/martinblech/xmltodict/blob/ae19c452ca000bf243bfc16274c060bf3bf7cf51/xmltodict.py#L198
+ */
+private class XmlToDictParseCall extends Decoding::Range, DataFlow::CallCfgNode {
+  XmlToDictParseCall() { this = xmltodict().getMember("parse").getACall() }
+
+  override predicate mayExecuteInput() { none() }
+
+  override DataFlow::Node getAnInput() { result = this.getArg(0) }
+
+  override DataFlow::Node getOutput() { result = this }
+
+  override string getFormat() { result = "JSON" }
+}
+
+// ---------------------------------------------------------------------------
+// ujson
+// ---------------------------------------------------------------------------
+/** Gets a reference to the `ujson` module. */
+API::Node ujson() { result = API::moduleImport("ujson") }
+
+/**
+ * A call to `ujson.loads`
+ * See https://pypi.org/project/ujson/#usage
+ */
+private class UltraJsonLoadsCall extends Decoding::Range, DataFlow::CallCfgNode {
+  UltraJsonLoadsCall() { this = ujson().getMember("loads").getACall() }
+
+  override predicate mayExecuteInput() { none() }
+
+  override DataFlow::Node getAnInput() { result = this.getArg(0) }
+
+  override DataFlow::Node getOutput() { result = this }
+
+  override string getFormat() { result = "JSON" }
+}
```

Comment:
This is now part of our standard modeling: https://github.com/github/codeql/blob/main/python/ql/src/semmle/python/frameworks/Ujson.qll

```suggestion
```